evaluation/tasks/mrpc-confirmation/mrpc-confirmation.py

```diff
@@ -0,0 +1,152 @@
+                        min_length=self.args.min_length,
+                        do_sample=self.args.do_sample,  # need to be set to true not to use greedy sampling
+                        early_stopping=self.args.early_stopping,
+                        # whether to stop when num_beams sentences are generated
+                        num_beams=self.args.num_beams,
+                        temperature=self.args.temperature,  # lower than 1 conservative, greater than one diverse
+                        top_k=self.args.top_k,
+                        # number of highest probability vocabulary tokens to keep for top-k-filtering
+                        top_p=self.args.top_p,  #
+                        repetition_penalty=self.args.repetition_penalty,
+                        length_penalty=self.args.length_penalty  # 1 no penalty >1 foster long sentences
+
+                    )
+                    decoded_output = self.tokenizer.decode(output[0], skip_special_tokens=True)
+                return decoded_output
+
+            paraphrase = get_output(sample)
+
+            # use the output to confirm
+            prompt = TEMPLATE_CONFIRMATION.render(
+                sent1=sample["sentence1"],
+                sent2=paraphrase
+            )
+            # Tokenize and construct this sample
+            inputs = self.tokenizer(
+                prompt,
+                return_tensors="pt",
+                truncation=True,
+            )
+
+            sample_confirmation = {
+                "prompt": prompt,
+                "input_ids": inputs["input_ids"],
+                "attention_mask": inputs["attention_mask"],
+                "input_len": inputs["attention_mask"].shape[1],
+            }
+            confirmation_output = get_output(sample_confirmation)
+
+            if is_first:
+                is_first = False
+                log_msg = "Evaluation example for MRPC-Negative\n"
+
+                log_msg += "\nprompt#1 (Standard):\n" + sample["prompt"]
+                log_msg += "\nmodel output:\n" + paraphrase
+
+                log_msg += "\n\nprompt#2 (Negative):\n" + sample_confirmation["prompt"]
+                log_msg += "\nmodel output:\n" + confirmation_output
+                logger.info(log_msg)
+
+            consistency += int(confirmation_output.lower() != "yes")
+
+            # log the prompts and the outputs
+            paraphrase_prompts.append(sample["prompt"])
+            confirmation_prompts.append(sample_confirmation["prompt"])
+
+            paraphrase_prompt_answers.append(paraphrase)
+            confirmation_prompt_answers.append(confirmation_output)
+
+        self.metrics = {
+            "consistency": consistency / len(dataset) * 100,
+            "para prompts": paraphrase_prompts,
+            "conf prompts": confirmation_prompts,
+            "para answers": paraphrase_prompt_answers,
+            "conf answers": confirmation_prompt_answers
```

Comment:
use full phrases in snbake case for readability 
```suggestion
            "paraphrase_prompts": paraphrase_prompts,
            "confirmation_prompts": confirmation_prompts,
            "paraphrase_prompt_answers": paraphrase_prompt_answers,
            "confirmation_prompt_answers": confirmation_prompt_answers
```